diff -uNr debian/control debian/control
--- debian/control	2012-11-30 15:43:24.000000000 -0600
+++ debian/control	2013-05-18 22:25:01.663929566 -0500
@@ -7,12 +7,10 @@
 Build-Depends: debhelper (>= 8),
                python-all (>= 2.6.6-3~),
                python-setuptools,
-               python-unittest2,
-               python3-all (>= 3.1.2-12~),
-               python3-setuptools
+               python-unittest2
 Build-Depends-Indep: libjs-jquery, python-sphinx (>= 1.0.7+dfsg-1~)
 Standards-Version: 3.9.3
-X-Python-Version: >= 2.4
+X-Python-Version: >= 2.6
 X-Python3-Version: >= 3.0
 Homepage: http://www.voidspace.org.uk/python/mock/
 Vcs-Svn: svn://svn.debian.org/python-modules/packages/python-mock/trunk/
diff -uNr debian/rules debian/rules
--- debian/rules	2012-11-30 15:26:47.000000000 -0600
+++ debian/rules	2013-05-18 17:41:21.769672613 -0500
@@ -4,15 +4,14 @@
 #export DH_VERBOSE=1
 
 PYVERS :=  $(shell pyversions -r)
-PY3VERS := $(shell py3versions -r)
 
 %:
-	dh $@ --with python2,python3,sphinxdoc
+	dh $@ --with python2,sphinxdoc
 	
 .PHONY: override_dh_auto_build
 override_dh_auto_build:
 	set -e; \
-	for py in $(PYVERS) $(PY3VERS); do \
+	for py in $(PYVERS); do \
 		$$py -B setup.py build; \
 	done
 	PYTHONPATH=. sphinx-build -b html -N docs/ docs/.build/html
@@ -20,7 +19,7 @@
 .PHONY: override_dh_auto_clean
 override_dh_auto_clean:
 	set -e; \
-	for py in $(PYVERS) $(PY3VERS); do \
+	for py in $(PYVERS); do \
 		$$py -B setup.py clean; \
 		rm -rf build; \
 	done
@@ -32,11 +31,6 @@
 		$$py -B setup.py install --skip-build --root debian/python-mock \
 		                      --install-layout deb; \
 	done
-	set -e; \
-	for py in $(PY3VERS); do \
-		$$py -B setup.py install --skip-build --root debian/python3-mock \
-		                      --install-layout deb; \
-	done
 	rm debian/python*-mock/usr/lib/python*/*-packages/mock*.egg-info/SOURCES.txt
 	sed -i 's/Metadata-Version: 1\.0/Metadata-Version: 1\.1/' debian/python-mock/usr/lib/python2.*/*-packages/mock*.egg-info/PKG-INFO
 
@@ -57,7 +51,7 @@
 override_dh_auto_test:
 ifeq (,$(filter nocheck,$(DEB_BUILD_OPTIONS)))
 	set -e; \
-	for py in $(PYVERS) $(PY3VERS); do \
+	for py in $(PYVERS); do \
 		$$py setup.py test ; \
 	done
 endif
